pkg/controller/packet/server/device/managed.go

```diff
@@ -30,10 +30,10 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	"github.com/hasheddan/stack-packet-demo/api/server/v1alpha1"
-	packetv1alpha1 "github.com/hasheddan/stack-packet-demo/api/v1alpha1"
-	packetclient "github.com/hasheddan/stack-packet-demo/pkg/clients"
-	devicesclient "github.com/hasheddan/stack-packet-demo/pkg/clients/device"
+	"github.com/hasheddan/stack-packet/apis/server/v1alpha1"
```

Comment:
Directory `api/` was renamed to `apis/` to ease ability to compare with existing stacks.